api/main.py

```diff
@@ -186,6 +188,48 @@ async def get_root_node(node_id: str):
     return node
 
 
+@app.get('/trigger_completed_event/{node_id}')
+async def trigger_completed_event(node_id: str, wait_time_hours: int = 0,
+                                  wait_time_minutes: int = 0,
+                                  wait_time_seconds: int = 0):
+    """Trigger an event when all child nodes are completed of
+       a given node"""
+    try:
+        nodes = await db.find_by_attributes(Node,
+                                            {"parent": ObjectId(node_id)})
+
+        if len(nodes) == 0:
```

Comment:
Wouldn't it be better to have here?:

```suggestion
        if not nodes:
```

Reply:
Yes, I think it is more readable.